m4opt/models/background/__init__.py

```diff
@@ -1,7 +1,8 @@
 from .core import Background
 from .zodiacal import ZodiacalBackground
+from .skybright import SkyBrightness
 
-__all__ = ('ZodiacalBackground',)
+__all__ = ('ZodiacalBackground', 'SkyBrightness',)
```

Comment:
Do we want to call this `SkyBackground`, for consistency with the other classes in this module?

Reply:
Yes, we should do that.